insights/parsers/kpatch_patches.py

```diff
@@ -0,0 +1,34 @@
+"""
+KpatchPatches - report locally stored kpatch patches
+====================================================
+
+This parser creates a list of the module names of locally
+stored kpatch modules. If no modules are installed, the
+list will be empty.
+
+"""
+
+from .. import parser, CommandParser
+from insights.specs import Specs
+from insights.core.plugins import ContentException
+from insights.parsers import SkipException
+
+
+@parser(Specs.kpatch_patch_files)
+class KpatchPatches(CommandParser):
+    """
+    A parser for getting modules names of locally stored kpatch-patch files
+    """
+
+    def __init__(self, context):
+        try:
+            super(type(self), self).__init__(context)
+        except ContentException:
```

Comment:
Thanks for updating this. Since the `__init__` of the superclasses will be called implicitly when it's omitted in the sub-class. It's OK to remove it here.

And `ContentException` looks more suitable here than the `SkipException`, since the `SkipException` will be discarded directly by the core, however, the `ContentException` is track-able. It would be useful for insights to do further check about e.g. if there are CUs using the "kpatch" in an abnormal way? (e.g. we cannot find the regular patches even the kpatch is installed)

How do you think?

Reply:
Sure, changed.